include/real/const_precision_iterator.hpp

```diff
@@ -359,7 +359,7 @@ namespace boost {
                                // If the explicit number didn't reaches the full precision (the end)
                                // then the number interval is defined by truncation.
 
-                               for(precision_t i = _precision; i < _precision + n; i++) {
+                               for(precision_t i = this->_precision; i < this->_precision + n; i++) {
```

Comment:
For good coding practice.
Also, change "i++" to "++i" for these loops in next commit.(A small performance improvement)